configure.ac

```diff
@@ -2,10 +2,10 @@ dnl Process this file with autoconf to produce a configure script.
 AC_INIT(udftools, 1.3, , , [https://github.com/pali/udftools/])
 AC_CONFIG_AUX_DIR(config)
 AM_CONFIG_HEADER(include/config.h:include/config.in)
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([subdir-objects])
```

Comment:
This change is breaking build parallelism of make (-j option) and cause that two parallel processes of make would overwrite e.g. mkudffs.o file.

Reply:
Clean-up was made here:  f2ce3c2